src/vip-framework/checks/checkIsolatedPoolsComptrollers.ts

```diff
@@ -71,20 +71,21 @@ const runPoolTests = async (pool: PoolMetadata, poolSupplier: string) => {
 
   for (const market of markets) {
     const marketData: MarketMetadata = await comptroller.markets(market);
-    if (marketData.collateralFactorMantissa.isZero() && !supplyMarket) continue;
-    if (!supplyMarket) {
+    if (!supplyMarket && !marketData.collateralFactorMantissa.isZero()) {
       supplyMarket = await ethers.getContractAt(VTOKEN_ABI, market, signer);
       supplyUnderlying = await ethers.getContractAt(ERC20_ABI, await supplyMarket.underlying(), signer);
       const balance = await supplyUnderlying.balanceOf(poolSupplier);
       if (balance.isZero()) {
         supplyMarket = undefined;
-        continue;
       }
-    } else if (!borrowMarket) {
+    }
+
+    if (!borrowMarket) {
```

Comment:
I think `borrowMarket` will be the `supplyMarket` in every case (because we have removed the `else` statement)